js/app.js

```diff
@@ -200,61 +201,83 @@ label_list.on('contextmenu', function (e) {
 });
 
 //edit label name
-label_list.on('dblclick', '.label', function () {
+$('#label-list').on('dblclick', '.label', function () {
   //enble editing
   $(this).children('.label-name')[0].contentEditable = true;
   //open textbox
   $(this).children('.label-name').focus().selectText();
 });
 
 // user pressed enter on label name change
-label_list.on('keypress', '.label-name', function (e) {
+$('#label-list').on('keypress', '.label-name', function (e) {
   if (e.which === 13) {
     $(this).blur();
```

Comment:
If you create a label and press enter without any text the label goes away which is good! but my console also throws an error to here(?) i don't know if you wanna check it or not. approved though! the behavior's still right

Reply:
hmmmm thats weird. couldnt really debug it since for some reason its happening at the return statement and originating in one of the jquery library files.

Reply:
ahhh wait there's more off here.  So if you delete the name in the label, it removes all the highlights, which is definitely not what we want

Reply:
has this been fixed or nah

Reply:
just pushed

Reply:
just pushed